api/src/client/DynamicsBusinessIdsClient.ts

```diff
@@ -0,0 +1,36 @@
+import { LogWriterType } from "@libs/logWriter";
+import axios, { AxiosError } from "axios";
+import { BusinessIdClient, BusinessIdResponse } from "../domain/types";
```

Comment:
[pebble] I think the `BusinessIdResponse` type should be defined in this file, not in domain... it defines the data structure for a Dynamics response type, right? That would be client-specific code, not our domain.